source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/index.tsx

```diff
@@ -56,6 +61,7 @@ class Table extends React.Component<IProps> {
                 activeProjectColumns,
                 activeProductColumns,
                 projectPath,
+                withOptions
```

Comment:
since we are propdrilling, this should probably be something more specific.
maybe `showProjectActions`?

Reply:
changed